Makefile

```diff
@@ -0,0 +1,9 @@
+design_concept:
+	typst watch SA/design_concept/design_concept.typ --root SA
+
+project_documentation:
+	typst watch SA/project_documentation/project_documentation.typ --root SA
+
+all:
+	typst compile SA/design_concept/design_concept.typ --root SA
```

Comment:
You can add an output path like this:

```
typst.exe compile [OPTIONS] <INPUT> [OUTPUT]
```

If we compiled them both into the same directory, we could upload that as artifact